server/auth/types/authentication_type.test.ts

```diff
@@ -106,4 +158,47 @@ describe('test tenant header', () => {
     const result = await dummyAuthType.authHandler(request, response, toolkit);
     expect(result.requestHeaders.securitytenant).toEqual('dummy_tenant');
   });
+
+  it(`keepalive should not shorten the cookie expiry`, async () => {
+    const keepAliveConfig = {
+      multitenancy: {
+        enabled: true,
+      },
+      auth: {
+        unauthenticated_routes: [] as string[],
+      },
+      session: {
+        keepalive: true,
+        ttl: 1000,
+      },
+    } as SecurityPluginConfigType;
+    const keepAliveDummyAuth = new DummyAuthType(
+      keepAliveConfig,
+      new BrowserSessionStorageFactory('security_cookie'),
+      router,
+      esClient,
+      coreSetup,
+      logger
+    );
+    const testCookie: SecuritySessionCookie = {
+      credentials: {
+        authHeaderValueExtra: true,
+      },
+      expiryTime: 2000,
+    };
+    // Set cookie
+    sessionStorage.setItem('security_cookie', JSON.stringify(testCookie));
+    const request = httpServerMock.createOpenSearchDashboardsRequest({
+      path: '/internal/v1',
+    });
+    const response = jest.fn();
+    const toolkit = {
+      authenticated: jest.fn((value) => value),
+    };
+    const _ = await keepAliveDummyAuth.authHandler(request, response, toolkit);
```

Comment:
remove `const _ =`?

Reply:
` _` Is a standard name for something that is unused. Since I am checking the results of the test via the cookie, and not through the result of the function, I named the value to `_` to indicate it's unused. I think its fine as is.

Reply:
hmm. it should be fine with just `await keepAliveDummyAuth.authHandler(request, response, toolkit);` without const declaration too. correct?

Reply:
Sure I don't think there is a difference between the two. I can do this if you prefer, but if you don't have any strong opinions I will keep it as is.

Reply:
if this is a pattern being already used i don't mind keeping it, else let's remove it

Reply:
Removed in: [278c40d](https://github.com/opensearch-project/security-dashboards-plugin/pull/1773/commits/278c40d295dba69c1dabfcda32b9f12cb321b765)